lib/Target/AArch64/AArch64MCInstLower.cpp

```diff
@@ -160,7 +160,8 @@ MCOperand AArch64MCInstLower::lowerSymbolOperandELF(const MachineOperand &MO,
       index = Printer.GetPOTIndex(MO.getIndex());
     }
     return MCOperand::createImm(index);
-  } else if (SourceFlag == AArch64II::MO_SEC) {
+  }
```

Comment:
These flags are mutually exclusive. I think it's better to keep the `else` since it's easy to forget to put it back if someone removes the `return`.